styles/prosilver/theme/ideas.css

```diff
@@ -93,3 +93,5 @@ dd.topics {
 .voteslist ul, .voteslist ul li { list-style: none; }
 
 .view-all, .view-all:hover { color: #536482; }
+
+#duplicatelink { margin: 0 5px 0 0; }
```

Comment:
Well I'd rather not create a whole class just to tweak this one element a little. FWIW, there are already other ID selectors in this CSS file anyways ;)